clients/ui/frontend/src/app/types.ts

```diff
@@ -22,7 +22,7 @@ export type ModelRegistry = {
 };
 
 // TODO: Change in the backend AND frontend to "items" instead of "model-registries"
-export type ModelRegistryResponse<T> = {
+export type ModelRegistryBody<T> = {
```

Comment:
Why change the name? And is the TODO comment above still applicable?

Reply:
No, great catch here, and yes, the change it's cause we are using it for both requests and responses